### PR TITLE
Improvements on password reset

### PR DESCRIPTION
**Pull request by viniciusld**

### CODE REVIEW

File: config/mail.php [hunk 1]
```diff
@@ -55,7 +55,8 @@
     |
     */
 
-    'from' => ['address' => null, 'name' => null],
+    // 'from' => ['address' => null, 'name' => null],
```
Comment (wesleyhf) [inline config/mail.php]: Remove this comment.

File: resources/views/auth/passwords/email.blade.php [hunk 2]
```diff
@@ -16,12 +16,14 @@
                 <form method="POST" action="{{ url('/password/email') }}">
```
Comment (wesleyhf) [inline resources/views/auth/passwords/email.blade.php]: Check if this route has a name.

File: resources/views/auth/passwords/reset.blade.php [hunk 3]
```diff
@@ -2,68 +2,46 @@
...
 @section('content')
 <div class="container">
-    <div class="row">
-        <div class="col-md-8 col-md-offset-2">
-            <div class="panel panel-default">
-                <div class="panel-heading">Reset Password</div>
+    <div class="mdl-grid">
+        <div class="mdl-cell mdl-cell--4-col-desktop mdl-cell--4-offset-desktop">
+            <h3 class="mdl-typography--headline">Reset Password</h3>
+            <form action="{{ url('/password/reset') }}" method="POST">
```
Comment (wesleyhf) [inline resources/views/auth/passwords/reset.blade.php]: Check if this route has a name.

File: resources/views/auth/passwords/email.blade.php (hunk 2 above)
Comment (viniciusld) [inline resources/views/auth/passwords/email.blade.php]: The auth routes doesn't have name

File: resources/views/auth/passwords/reset.blade.php (hunk 3 above)
Comment (viniciusld) [inline resources/views/auth/passwords/reset.blade.php]: The auth routes doesn't have name

**Comment by wesleyhf** [comment]
@dbpolito

**Pull request merged**